### DISCUSSION TITLE
Should changing the view range with an open report update that report?

### DISCUSSION THREAD

**Discussion by ethyoo** [firefly-iii/firefly-iii#8898] [Firefly III] [answered]
With a report open: Using the drop-down menu to change the view range does not update the report with the new date range. Changing the view range to "Everything" does still show a warning that, in my case, "630 days of data may take a while to load." The report itself does not change.

I'm not sure if this behavior is expected.

I know I can manually change the URL (e.g., using [magic words](https://docs.firefly-iii.org/how-to/firefly-iii/finances/reports/#url-options)).

**Comment by JC5** [comment, marked as answer]
This is expected. Not all views use the date range at the top.

**Reply by ethyoo** [reply]
Thanks for the prompt response!
